pkg/controllers/raycluster_controller.go

```diff
@@ -154,6 +156,11 @@ func (r *RayClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		logger.Error(err, "Failed to update OAuth ClusterRoleBinding")
 	}
 
+	_, err = r.kubeClient.NetworkingV1().NetworkPolicies(cluster.Namespace).Apply(ctx, desiredNetworkPolicy(&cluster), metav1.ApplyOptions{FieldManager: controllerName, Force: true})
+	if err != nil {
+		logger.Error(err, "Failed to update NetworkPolicy")
+	}
```

Comment:
shouldn't the error be returned? I see the same is done for the previous objects (ClusterRoleBindings, ServiceAccounts, etc)

Reply:
Just reviewing this and maybe the comment got moved around but was this addressed?

Reply:
I was planning to open another PR that fixes all the instances of this issue once this is merged